inter/block.go

```diff
@@ -15,12 +16,33 @@ type Block struct {
 	SkippedTxs  []uint32      // indexes of skipped txs, starting from first tx of first event, ending with last tx of last event
 	GasUsed     uint64
 	Root        hash.Hash
+	prevRandao  common.Hash
 }
 
 func (b *Block) EstimateSize() int {
 	return (len(b.Events)+len(b.InternalTxs)+len(b.Txs)+1+1)*32 + len(b.SkippedTxs)*4 + 8 + 8
 }
 
+func (b *Block) SetPrevRandao(prevrandao common.Hash) {
+	b.prevRandao = prevrandao
+}
+
+func (b *Block) GetPrevRandao() common.Hash {
+	return b.prevRandao
+}
+
+// ComputePrevRandao computes the prevRandao from event hashes.
+func ComputePrevRandao(events []hash.Event) common.Hash {
```

Comment:
nit: should this function be in the gossip package in its own file, since it is only used by `c_block_callbacks.go`?

Reply:
I thought about the place for this func. Yeah, `gossip` package is the right place because of the reason you mentioned.